CMakeLists.txt

```diff
@@ -90,6 +90,20 @@ if(CERES_FOUND)
   include_directories(${CERES_INCLUDE_DIRS})
 endif(CERES_FOUND)
 
+find_package(HDF5 1.8 REQUIRED COMPONENTS CXX)
+
+if(HDF5_FOUND)
+  message("-- HDF5 found for checkpointing")
```

Comment:
CMake already prints a message that hdf5 was found, so why do we need two?

Reply:
Fair enough, deleted.